homeassistant/components/generic/strings.json

```diff
@@ -5,6 +5,10 @@
       "unknown": "[%key:common::config_flow::error::unknown%]",
       "already_exists": "A camera with these URL settings already exists.",
       "unable_still_load": "Unable to load valid image from still image URL (e.g. invalid host, URL or authentication failure).  Review log for more info.",
```

Comment:
In the future you could consider updating this message too -- though i can see looking at https://www.python-httpx.org/exceptions/ it still looks broad. I think it now means things like a network error, or error decoding the request, or a bad proxy? All of which are fairly broad.  Could do in a follow up PR if you think it is worthwhile.

Reply:
Yes could do.

I suppose this might apply in general to other integrations. Essentially anything where the user enters a URL is likely to have these sorts of problems.

Maybe there is some benefit to having a generalised helper that other integrations could use, turning the error into a message friendly to the target HA user base.